### DISCUSSION TITLE
Two separate applications from nicegui linked together.

### DISCUSSION THREAD

**Discussion by Kowersky** [zauberzeug/nicegui#4127] [Q&A] [answered]
### Question

I would like to make one application in which I can enter data and in the second application I would like it to be the same only cow the first. Just so they are connected to each other. That in one application I change then in the 2nd application is the same as in the first and vice versa. Please help because I have absolutely no idea how I could get this to work.

**Comment by rodja** [comment, marked as answer]
We are almost ready to release https://github.com/zauberzeug/nicegui/pull/4074 which allows you to do this via `app.storage`. Would that work for you?

**Reply by Kowersky** [reply]
> We are almost ready to release #4074 which allows you to do this via `app.storage`. Would that work for you?

Thanks, its working. Its posibble to set two diffrents var's for user?

**Reply by rodja** [reply]
Sorry, I do not understand your question. The storage objects are dictionaries. So they can hold multiple variables, yes.
